src/lib.js

```diff
@@ -677,6 +676,40 @@ var program,
       }
     },
 
+    /**
+     * Identifies whether the input data string is a graphql query or not
+     *
+     * @param {string} dataString - Input data string to check if it is a graphql query
+     * @returns {Object} - { result: true, graphql: {Object} } if dataString is a graphql query else { result: false }
+    */
+    identifyGraphqlRequest: function (dataString) {
+      try {
+        const rawDataObj = _.attempt(JSON.parse, dataString.replace(/\r\n/g, ''));
```

Comment:
@aman-v-singh Why do we need to replace `\r\n`? JSON.parse should take care of such characters.

Reply:
@VShingala 2 Reasons:-
- I was facing this error. It seems that [I need to escape the `\`](https://stackoverflow.com/questions/42068/how-do-i-handle-newlines-in-json). 
![image](https://github.com/postmanlabs/curl-to-postman/assets/121886615/530e60d4-ef96-4a55-b7a2-d96bf347f4cf)

- Postman App expects the query to be without any such characters.

Reply:
@aman-v-singh Hmm, In that case if we remove `/n` newline chars than wouldn't this also remove the correct newline from query as well? Which would be needed for better indentation?

Reply:
@VShingala The Postman app's graphql editor handles the indentation well based on the space.
eg:- 
This is the result obtained from curl to postman.
<img width="980" alt="Screenshot 2023-06-26 at 1 42 49 PM" src="https://github.com/postmanlabs/curl-to-postman/assets/121886615/6e2eb0b9-34e7-4024-81d4-dc14995c3045">
On importing the same curl code snippet we get the correct indentation.

<img width="1438" alt="Screenshot 2023-06-26 at 1 45 48 PM" src="https://github.com/postmanlabs/curl-to-postman/assets/121886615/946a8881-c8f1-4fdc-9818-e6e4b95df4e0">